manifests/templates/reconciler-manager-configmap.yaml

```diff
@@ -132,6 +132,10 @@ data:
          - name: gcenode-askpass-sidecar
            image: gcr.io/config-management-release/gcenode-askpass-sidecar:v1.0.4
            args: ["--port=9102", "--logtostderr"]
+           resources:
+             requests:
+               cpu: 10m
+               memory: 100Mi
```

Comment:
Does askpass need this much resource? The default memory request before this change was 10Mi.

Reply:
ran a few more benchmark and the highest i've seen is 16Mi. I have created a PR to reduce the default to 20Mi https://github.com/GoogleContainerTools/kpt-config-sync/pull/835

```
CPU(cores)   MEMORY(bytes)
7m           16Mi
```